src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -72,6 +73,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case HelpCommand.COMMAND_WORD:
             return new HelpCommand();
 
+        case ClearAllCommand.COMMAND_WORD:
+            return new ClearAllCommand();
+
```

Comment:
Place this in alphabetical order if possible, in the same order as the import statements